pkg/services/guardian/guardian.go

```diff
@@ -22,7 +22,14 @@ type DashboardGuardian interface {
 	CanAdmin() (bool, error)
 	HasPermission(permission models.PermissionType) (bool, error)
 	CheckPermissionBeforeUpdate(permission models.PermissionType, updatePermissions []*models.DashboardAcl) (bool, error)
+
+	// GetAcl returns ACL.
 	GetAcl() ([]*models.DashboardAclInfoDTO, error)
+
+	// GetAclWithoutDuplicates returns ACL and strips any permission
+	// that already has an inherited permission with higher or equal
+	// permission.
+	GetAclWithoutDuplicates() ([]*models.DashboardAclInfoDTO, error)
```

Comment:
Nit, but initialisms should be in consistent case in Go:
```suggestion
	// GetACLWithoutDuplicates returns ACL and strips any permission
	// that already has an inherited permission with higher or equal
	// permission.
	GetACLWithoutDuplicates() ([]*models.DashboardAclInfoDTO, error)
```